db/fixtures/ae_datastore/ManageIQ/ConfigurationManagement/AnsibleTower/Operations/StateMachines/Job.class/__methods__/launch_ansible_job.rb

```diff
@@ -35,13 +35,14 @@ def ansible_vars_from_objects(object, ext_vars)
   end
 
   def object_vars(object, ext_vars)
-    object.attributes.each_with_object(ext_vars) do |(key, value), hash|
+    key_list = object.attributes.keys.select { |k| k.start_with?('param', 'dialog_param') }
```

Comment:
Can you add a comment explaining why we're traversing the attributes twice?  Someone will ask and have no idea that we're in DRb land dealing with complex Hashes with default procs that don't work like normal Hashes on the DRb side.